SECURITY.md

```diff
@@ -8,20 +8,12 @@ The public repositories MUST NOT leak any valid private credential (e.g., a work
 
 ## Supported Versions
 
-Use this section to tell people about which versions of your project are
-currently being supported with security updates.
+This product is being made available as a 0.4 version and is in active development at that version.
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 5.1.x   | :white_check_mark: |
-| 5.0.x   | :x:                |
-| 4.0.x   | :white_check_mark: |
-| < 4.0   | :x:                |
+| 0.4   | :white_check_mark: |
```

Comment:
One of the commit messages says v 0.5....

Reply:
yes... that was arbitrary (and probably wrongly indicative of our level of progress) and I'm hoping it will get squashed ...

I think 0.4 would be better.

Reply:
We can squash and merge this first PR.  I usually advocate for rebasing to keep development history in tact as much as possible.  This puts the squash/rebase/history editing in the hands of the developer issuing PR's.  Just let me know if we should squash this PR once you get all the commits in it you want.